apps/charterafrica/scripts/ecosystem.js

```diff
@@ -0,0 +1,21 @@
+require("dotenv").config({ path: "./.env" });
+require("dotenv").config({ path: "./.env.local" });
+
+async function main() {
+  const { RESOURCES_SECRET_TOKEN } = process.env;
+  const { NEXT_PUBLIC_APP_URL } = process.env;
+  const method = process.argv[2];
+  const headers = {
+    "x-api-key": RESOURCES_SECRET_TOKEN,
+  };
+  const res = await fetch(
+    `${NEXT_PUBLIC_APP_URL}/api/v1/resources/ecosystem/entities`,
+    {
+      method,
+      headers,
+    },
+  );
+  return res.json();
+}
+
+main().then(console.log);
```

Comment:
Since you're using `async`, why not `await`?

```suggestion
const responseJson = await main()
console.log(responseJson)
```

Reply:
@kilemensi do we support top level await yet?

Reply:
It [is](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#browser_compatibility) in modules @koechkevin ... you may have to save the file as `ecosystem.mjs`.